__DOC_RESOURCES__/customStyles.css

```diff
@@ -6,6 +6,52 @@ Custom CSS for the JSDOC template
   display:none;
 }
 
+/*body {*/
```

Comment:
supposed to have a comment?

Reply:
Yea, for now, we're still debating the right CSS changes.